gen/utils/data/class.go

```diff
@@ -83,12 +89,41 @@ const (
 	Cloud
 )
 
-type Child struct {
-	// The name of the child class, ex "fvTenant".
-	// This is used as the key for the map of all classes.
-	ClassName string
-	// When it is a relationship class, this is the class to which the relationship points.
-	PointsToClass string
+type Relation struct {
+	// The class from which the relationship is defined.
+	FromClass string
+	// Indicates if _from_ should be included in the resource name.
+	//  ex. "fvRsBdToOut" would be "data_source_aci_relation_from_bridge_domain_to_l3_outside".
+	IncludeFrom bool
+	// Indicates if the class is a relational class.
+	RelationalClass bool
+	// The class to which the relationship points.
+	ToClass string
+	// The type of the relationship.
+	Type RelationhipTypeEnum
+}
+
+// The enumeration options of the relationship type.
+type RelationhipTypeEnum int
```

Comment:
```suggestion
type RelationshipTypeEnum int
```
Make the change everywhere